core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -528,25 +527,23 @@ var ExplorationEditorMainTab = function() {
   // ---- HINT ----
 
   this.addHint = async function(hint) {
-    await addHintButton.click();
+    await action.click('Add hint button', addHintButton);
     var addHintModal = element(
       by.cssContainingText('.protractor-test-hint-modal', 'Add Hint'));
     await waitFor.visibilityOf(
       addHintModal, 'Add hint modal takes too long to appear');
-    await element(by.css('.protractor-test-hint-text')).all(by.tagName('p'))
-      .last().click();
+    var hintButton = await element(
+      by.css('.protractor-test-hint-text')).all(by.tagName('p')).last();
+    await action.click('Hint Button', hintButton);
```

Comment:
Hmm... I'm not sure whether this is a button? Can you please check what this code is doing and add a proper variable and element name?